virt/live_migration/virt-migrate-vmi.adoc

```diff
@@ -9,6 +9,11 @@ toc::[]
 
 Manually initiate a live migration of a virtual machine instance to another node using either the web console or the CLI.
 
+[NOTE]
+====
+You can perform live migration of virtual machines between nodes if the virtual machine CPUs are identical. If the virtual machine CPUs are incompatible, live migration between the nodes is prevented.
```

Comment:
  ... if the virtual machine CPUs are identical. ...

This one sounds incorrectly. Migration allowed when there is another node in a cluster which also supports current CPU. But when all other nodes (expect where VM is running) do not support such CPU - migration will be prevented.

Reply:
Agreed that this should be rephrased. The discussion needs to be about the host's CPUs and if they can emulate the requested type. We also are missing a key element of the messaging: this is about VMs using host-model CPUs. Suggest something like:

If a virtual machine uses a host model CPU, you can perform live migration of that virtual machine between nodes that support that CPU model. Migration to nodes that do not support the virtual machine's CPU model will not be allowed.

@iholder-redhat please weigh in.